test/ggsn_proxy_SUITE.erl

```diff
@@ -835,6 +843,14 @@ end_per_testcase(delete_bearer_requests_multi, Config) ->
     ok = meck:delete(ergw_gtp_c_socket, send_request, 7),
     end_per_testcase(Config),
     Config;
+end_per_testcase(TestCase, Config)
+  when TestCase == proxy_context_selection;
+       TestCase == proxy_context_invalid_selection;
+       TestCase == proxy_context_invalid_mapping;
+       TestCase == proxy_api_v2 ->
+    ok = meck:unload(gtp_proxy_ds),
+    end_per_testcase(Config),
+    Config;
```

Comment:
I think the return value is ignored (apart from special tagged tuples). So just 'ok' is a good return value.

Reply:
yes, but all end_per_testcase function currently do return config. No need to change that now.